generated/nidaqmx/base_interpreter.py

```diff
@@ -0,0 +1,1630 @@
+            self, counter, name_to_assign_to_channel="",
+            source_terminal="OnboardClock", min_val=1000, max_val=1000000):
+        pass
+
+    def add_ci_pulse_chan_time(
+            self, counter, name_to_assign_to_channel="", min_val=0.000001,
+            max_val=0.001, units=TimeUnits.SECONDS):
+        pass
+
+    def add_ci_pulse_width_chan(
+            self, counter, name_to_assign_to_channel="", min_val=0.000001,
+            max_val=0.1, units=TimeUnits.SECONDS, starting_edge=Edge.RISING,
+            custom_scale_name=""):
+        pass
+
+    def add_ci_semi_period_chan(
+            self, counter, name_to_assign_to_channel="", min_val=0.000001,
+            max_val=0.1, units=TimeUnits.SECONDS, custom_scale_name=""):
+        pass
+
+    def add_ci_two_edge_sep_chan(
+            self, counter, name_to_assign_to_channel="", min_val=0.000001,
+            max_val=1.0, units=TimeUnits.SECONDS, first_edge=Edge.RISING,
+            second_edge=Edge.FALLING, custom_scale_name=""):
+        pass
+
+    def add_co_pulse_chan_freq(
+            self, counter, name_to_assign_to_channel="",
+            units=FrequencyUnits.HZ, idle_state=Level.LOW, initial_delay=0.0,
+            freq=1.0, duty_cycle=0.5):
+        pass
+
+    def add_co_pulse_chan_ticks(
+            self, counter, source_terminal, name_to_assign_to_channel="",
+            idle_state=Level.LOW, initial_delay=0, low_ticks=100,
+            high_ticks=100):
+        pass
+
+    def add_co_pulse_chan_time(
+            self, counter, name_to_assign_to_channel="",
+            units=TimeUnits.SECONDS, idle_state=Level.LOW, initial_delay=0.0,
+            low_time=0.01, high_time=0.01):
+        pass
+
+    def add_di_chan(
+            self, lines, name_to_assign_to_lines="",
+            line_grouping=LineGrouping.CHAN_FOR_ALL_LINES):
+        pass
+
+    def add_do_chan(
+            self, lines, name_to_assign_to_lines="",
+            line_grouping=LineGrouping.CHAN_FOR_ALL_LINES):
+        pass
+
+    def add_global_chans_to_task(self):
+        pass
+
+    @staticmethod
+    def add_lin_scale(
+            self, name, slope, y_intercept=0.0,
+            pre_scaled_units=UnitsPreScaled.VOLTS, scaled_units=None):
+        pass
+
+    @staticmethod
+    def add_map_scale(
+            self, name, prescaled_min, prescaled_max, scaled_min, scaled_max,
+            pre_scaled_units=UnitsPreScaled.VOLTS, scaled_units=None):
+        pass
+
+    @staticmethod
+    def add_network_device(self):
+        pass
+
+    @staticmethod
+    def add_polynomial_scale(
+            self, name, forward_coeffs, reverse_coeffs,
+            pre_scaled_units=UnitsPreScaled.VOLTS, scaled_units=None):
+        pass
+
+    @staticmethod
+    def add_table_scale(
+            self, name, prescaled_vals, scaled_vals,
+            pre_scaled_units=UnitsPreScaled.VOLTS, scaled_units=None):
+        pass
+
+    @staticmethod
+    def add_task(self):
+        pass
```

Comment:
As discussed previously, I think it would make more sense for the interpreter method names to follow the C API, like `create_task`: https://github.com/ni/nidaqmx-python/pull/210/files#r1111067299

I see this class has `clear_task` instead of `close`, but it still changes `create` to `add`.

Reply:
Got it. I have added a new function `get_all_functions()` which uses the `function_name` from the metadata directly.